spec/system/teacher_interface/authentication_spec.rb

```diff
@@ -189,26 +189,26 @@ def when_i_click_save_and_sign_out
   end
 
   def then_i_see_the_sign_up_form
```

Comment:
There's also a helper for this although it only checks the page has loaded (`then_i_see_the(:sign_up_form)`). If you want to check the page content then this is probably still the way to go.

Reply:
I would vote to keep it with checking the page content as that would make it slightly more extensive. But, i'm not fussed either way.